com.hahoto.onelimx.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.hahoto.onelimx
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: 1LimX
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+modules:
+  - name: 1LimX
+    # There are other types of buildsystems, like autotools, cmake, cmake-ninja, simple, qmake
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    sources:
+      - type: file
+        url: https://www.dropbox.com/scl/fi/fli2wzrhyslc2vs8hnpfp/1LimX-Linux-v1.2.10-amd64-setup.deb?rlkey=fnhru84c89qm6fq8erjdox9v4&dl=1
+        dest: 1LimX
+        sha256: bf8020c4cb9a5cdccf992c3209097d719d3df46f9a397d39c82731a886737484
+        # Automatically check for updates and create merge requests
+        x-checker-data:
+          # There are different types of checkers, see the documentation for more information
+          type: anitya
+          # This is the ID of the project on anitya
+          project-id: 1
```

Comment:
This belongs to a separate project https://release-monitoring.org/project/1 ?

Reply:
Hi @bbhtt,

Does that mean I need to push my project to [https://release-monitoring.org](https://release-monitoring.org/) to get the id?

Reply:
No, use the snap source add add a snap checker. You cannot just publish it to release-monitoring.